images/clustermesh-apiserver/Dockerfile

```diff
@@ -49,6 +58,11 @@ RUN apt-get update && apt-get install -y binutils-aarch64-linux-gnu binutils-x86
 RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium --mount=target=/root/.cache,type=cache --mount=target=/go/pkg,type=cache \
     ./build-gops.sh
 
+# BUILDPLATFORM is an automatic platform ARG enabled by Docker BuildKit.
+# Represents the plataform where the build is happening, do not mix with
```

Comment:
typo: plataform -> platform

Reply:
Removed this comment, as it shouldn't be `BUILDPLATFORM` anyway.